mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -440,6 +440,21 @@ func resourceContainerCluster() *schema.Resource {
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
 									<% unless version == 'ga' -%>
+									"image_type": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Default:     "cos",
```

Comment:
Is this an API-set default?

Reply:
Yes, I set the default value for Container-Optimized OS with docker.

Reply:
I guess I'm wondering if this is the value returned by the API if `image_type` is not set? If we're going to have a default, the default API-returned value is probably the default that we want here. If that's is `cos`, great, I just saw [this documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#cos) yesterday that mentions that `cos` is now deprecated with GKE node version `1.19`, so I wanted to make sure.

Reply:
Yes, you're right, Yesterday I received an email from google that they announced that the container optimized with docker would be deprecated. These changes were for before that announcement, so due to this announcement, I suppose it would be better to set the `cos_containerd` as a default value.

Reply:
Sorry for my delay, I guess my question would be - does it need a default? And/or what is the current default in the API. Like, if you don't set an `image_type` does the API return a value there, or can it be blank?

Reply:
@megan07 No problem, as I tested the Node Auto-provisioning using gcloud CLI by default, the API set the image_type to `cos`, and in the meantime, this type of image will be deprecated, so I suppose it would be better to set the default value from the terraform perspective.

Reply:
Got it. Ok, I think that sounds reasonable. Thanks for the background there!

Reply:
@megan07 Thank you for running the tests, and I can not see the test result to find out what's the problem or should I do any modification in which part of the code.